fixchain/fix_and_log_test.go

```diff
@@ -192,15 +195,23 @@ var newFixAndLogTests = []fixAndLogTest{
 
 func TestNewFixAndLog(t *testing.T) {
 	// Test that expected chains are logged when adding a chain using QueueChain()
+	ctx := context.Background()
 	for i, test := range newFixAndLogTests {
 		seen := make([]bool, len(test.expLoggedChains))
 		errors := make(chan *FixError)
-		client := &http.Client{Transport: &testRoundTripper{t: t, test: &test, testIndex: i, seen: seen}}
-		fl := NewFixAndLog(1, 1, errors, client, client, test.url, newNilLimiter(), false)
+		c := http.Client{Transport: &testRoundTripper{t: t, test: &test, testIndex: i, seen: seen}}
```

Comment:
Why remove the '&' to then only have to pass in c as '&c' everywhere it's used?  Why not keep c defined as a pointer?

Reply:
Done; I think comes from a residual C++-based fear of taking the address of a temporary.